P5/Source/Specs/citeStructure.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="header" ident="citeStructure">
+  <gloss versionDate="2020-02-03" xml:lang="en">citation structure</gloss>
+  <desc versionDate="2020-02-03" xml:lang="en">declares a structure and method for citing the current document.</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="att.citeStructurePart"/>
+  </classes>
+  <content>
+    <sequence>
+      <elementRef key="citeData" minOccurs="0" maxOccurs="unbounded"/>
+      <elementRef key="citeStructure" minOccurs="0" maxOccurs="unbounded"/>
+    </sequence>
+  </content>
+  <attList>
+    <attDef ident="delim" usage="opt">
```

Comment:
`@delim` is only optional on the outermost `<citeStructure>`, otherwise it's impossible to parse the input, right? One might think of using different character sets for the `@use` attribute (to construct a citation string like "A3µ" without any delimiters) but I think that's even more complicated?

So, I think what I'm trying to say is that we need a Schematron constraint that asserts `@delim` on nested `<citeStructure>`s

Reply:
Agreed. Will do.